src/github/credentials.ts

```diff
@@ -177,6 +192,18 @@ export class CredentialStore {
 		statusBarItem.command = command;
 	}
 
+	private willStartLogin(authority: string): void {
+		const status = this._authenticationStatusBarItems.get(authority);
+		status.text = `$(mark-github) Signing in to ${authority}...`;
+		status.command = AUTH_INPUT_TOKEN_CMD;
```

Comment:
it seems sort of strange to me to start a different method of completing auth when auth is in flight.

I think I would expect clicking the status bar item to not do anything while it has that text

Reply:
It... is a bit odd, I won't deny that 😅

I initially made it a popup with an `Enter token manually` button. The trouble is, there's no way to destroy those without the user interacting with them. So in the typical case where auth succeeds, it's left lying around, which felt janky to me.

This isn't meant to be particularly discoverable. It's a companion to [this auth server PR](https://github.com/editor-tools/authentication-server/pull/13), which gives folks instructions on what to do if the URI handler doesn't work.